tests/foreman/api/test_discoveredhost.py

```diff
@@ -356,63 +262,6 @@ def test_positive_provision_pxe_less_host(self):
         :CaseImportance: Critical
         """
 
-    @pytest.mark.run_in_one_thread
```

Comment:
I would rather keep the test stubbed.
@lhellebr, do we plan to test Discovery on Libvirt?

Reply:
So my plan was to convert all of the Libvirt Discovery tests to the new test class I added `TestDiscoveredHost`. All of the helpers we used for Libvirt Discovery no longer work. See https://github.com/SatelliteQE/robottelo/pull/10187 for how I want to clean up the module.

Reply:
I see that you plan to test Discovery on RHV. Does it make sense to test Discovery also on libvirt? I tagged @lhellebr as he spun one env-libvirt instance up a few months back if I am not mistaken.

Reply:
I don't think so, but I am interested to see what Lukas thinks.

Reply:
I can't see how the change to underlaying virtualization adds any value to discovery tests.